README.md

```diff
@@ -8,6 +8,7 @@
 
 [![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/46214)
 ![CI Status](https://github.com/rust-marker/marker/actions/workflows/rust_bors.yml/badge.svg)
+[![License: MIT OR Apache-2.0](https://img.shields.io/crates/l/cargo_marker.svg)](#license)
```

Comment:
This label is only showing the MIT license in the label, could you check on this?

Reply:
Yep, yep, yep, it takes the license information from the place holder crate from marker-crate-reservations, which is MIT only, that why I asked you to update it, see the PR description, and my comment later

Reply:
Good to know, then I understand why this is blocked. Thank you!